### PR TITLE
Remove enum 2 macros

### PR DESCRIPTION
**Pull request by dcristoloveanu**
Remove enum 2 macros

### CODE REVIEW

File: inc/macro_utils/macro_utils.h [hunk 1]
```diff
@@ -238,20 +236,6 @@ typedef struct ENUM_VALUE_AND_STRING_TAG
 }ENUM_VALUE_AND_STRING;
 
 #define MU_DEFINE_ENUM_VALUE_AND_STRING(enumerationConstant, constantExpression) {enumerationConstant, MU_TOSTRING(enumerationConstant)},
```
Comment (mattdurak) [inline inc/macro_utils/macro_utils.h:238]: looks like it's not used anymore

File: inc/macro_utils/macro_utils.h [hunk 2]
```diff
@@ -238,20 +236,6 @@ typedef struct ENUM_VALUE_AND_STRING_TAG
 }ENUM_VALUE_AND_STRING;
```
Comment (mattdurak) [inline inc/macro_utils/macro_utils.h:236]: also not used from what I can tell

File: inc/macro_utils/macro_utils.h (hunk 1 above)
Comment (dcristoloveanu) [reply inc/macro_utils/macro_utils.h:238]: Love it, removed!

---
In reply to: [636327543](https://github.com/Azure/macro-utils-c/pull/75#discussion_r636327543) [](ancestors = 636327543)

File: inc/macro_utils/macro_utils.h (hunk 2 above)
Comment (dcristoloveanu) [reply inc/macro_utils/macro_utils.h:236]: Also removed.

---
In reply to: [636327914](https://github.com/Azure/macro-utils-c/pull/75#discussion_r636327914) [](ancestors = 636327914)

**Pull request merged**